content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -114,6 +114,20 @@ to the name of the namespace. You can't change this label's value.
 This is useful if you want to target a specific namespace with a label
 {{< glossary_tooltip text="selector" term_id="selector" >}}.
 
+### kubernetes.io/limit-ranger
+
+Example: `kubernetes.io/limit-ranger: "LimitRanger plugin set: cpu, memory request for container nginx; cpu, memory limit for container nginx"`
+
+Used on: Pod
+
+Kubernetes by default doesn't provide any resource limit, that means unless you explicitly define limits,
+your container can consume unlimited CPU and memory.
+You can define a default request or default limit for pods. You do this by creating a LimitRange in the relevant namespace.
+Pods deployed after you define a LimitRange will have these limits applied to them.
+The annotation `kubernetes.io/limit-ranger` records that resource defaults were specified for the Pod,
+and they were applied successfully.
+For more details, read about [LimitRanges](/docs/concepts/policy/limit-range).
```

Comment:
Ah. I suspect that this annotation might also be applied on a PersistentVolumeClaim. If so, we ought to add that detail here before we merge it.

@SergeyKanzhelev do you have an opinion here?

Reply:
yes, I noticed that PVC was added to the document as a quick fix. I'd prefer we address it as a separate PR.